src/main/java/uk/gov/pay/directdebit/payments/dao/PaymentViewSearchParams.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.pay.directdebit.payments.dao;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.Optional;
+
+public class PaymentViewSearchParams {
+
+    private Long page;
+    private Long displaySize;
+    private Optional<Pair<Long, Long>> paginationParams;
+
+    public PaymentViewSearchParams(Long page, Long displaySize) {
+        this.page = page;
+        this.displaySize = displaySize;
+    }
+
+    public Long getPage() {
+        return page;
+    }
+
+    public Long getDisplaySize() {
+        return displaySize;
+    }
+
+    public Optional<Pair<Long, Long>> getPaginationParams() {
```

Comment:
See discussions above regarding pair and optional. This can just be: `public PaginationParams getPaginationParams()`. Then, `paginationParams.getMaxSize()` or whatever, can be null and will be null checked wherever used